cmd/image-builder/config.go

```diff
@@ -255,6 +255,18 @@ func loadGithubActionsGitState() (GitStateConfig, error) {
 			JobType:         "postsubmit",
 			BaseCommitSHA:   *payload.HeadCommit.ID,
 		}, nil
+	case "workflow_dispatch":
+		var payload github.WorkflowDispatchEvent
+		err = json.Unmarshal(data, &payload)
+		if err != nil {
+			return GitStateConfig{}, fmt.Errorf("failed to parse event payload: %s", err)
+		}
+		return GitStateConfig{
+			RepositoryName:  *payload.Repo.Name,
+			RepositoryOwner: *payload.Repo.Owner.Login,
+			JobType:         "postsubmit",
+			BaseCommitSHA:   os.Getenv("GITHUB_SHA"),
```

Comment:
**GITHUB_SHA** can hold commit of branch or tag. https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch

We should read **GITHUB_REF** variable value as well. Write this information to the logs and pass to the oci-image-builder as parameter. Even if this data will not be used for making any decision in the tool, it will provide more information about execution context. This will be useful when troubleshooting.

Reply:
I've added BaseCommitRef field.